commandapi-core/src/main/java/dev/jorel/commandapi/wrappers/WrapperStringReader.java

```diff
@@ -0,0 +1,198 @@
+        return stringReader.read();
+    }
+
+    public void skip() {
+        stringReader.skip();
+    }
+
+    public static boolean isAllowedNumber(char c) {
+        return StringReader.isAllowedNumber(c);
+    }
+
+    public static boolean isQuotedStringStart(char c) {
+        return StringReader.isQuotedStringStart(c);
+    }
+
+    public void skipWhitespace() {
+        stringReader.skipWhitespace();
+    }
+
+    public int readInt() throws WrapperCommandSyntaxException {
+        try {
+            return stringReader.readInt();
+        } catch (CommandSyntaxException e) {
+            throw new WrapperCommandSyntaxException(e);
+        }
+    }
+
+    public long readLong() throws WrapperCommandSyntaxException {
+        try {
+            return stringReader.readLong();
+        } catch (CommandSyntaxException e) {
+            throw new WrapperCommandSyntaxException(e);
+        }
+    }
+
+    public double readDouble() throws WrapperCommandSyntaxException {
+        try {
+            return stringReader.readDouble();
+        } catch (CommandSyntaxException e) {
+            throw new WrapperCommandSyntaxException(e);
+        }
+    }
+
+    public float readFloat() throws WrapperCommandSyntaxException {
+        try {
+            return stringReader.readFloat();
+        } catch (CommandSyntaxException e) {
+            throw new WrapperCommandSyntaxException(e);
+        }
+    }
+
+    public static boolean isAllowedInUnquotedString(char c) {
+        return StringReader.isAllowedInUnquotedString(c);
+    }
+
+    public String readUnquotedString() {
+        return stringReader.readUnquotedString();
+    }
+
+    public String readQuotedString() throws WrapperCommandSyntaxException {
+        try {
+            return stringReader.readQuotedString();
+        } catch (CommandSyntaxException e) {
+            throw new WrapperCommandSyntaxException(e);
+        }
+    }
+
+    public String readStringUntil(char terminator) throws WrapperCommandSyntaxException {
+        try {
+            return stringReader.readStringUntil(terminator);
+        } catch (CommandSyntaxException e) {
+            throw new WrapperCommandSyntaxException(e);
+        }
+    }
+
+    public String readString() throws WrapperCommandSyntaxException {
+        try {
+            return stringReader.readString();
+        } catch (CommandSyntaxException e) {
+            throw new WrapperCommandSyntaxException(e);
+        }
+    }
+
+    public boolean readBoolean() throws WrapperCommandSyntaxException {
+        try {
+            return stringReader.readBoolean();
+        } catch (CommandSyntaxException e) {
+            throw new WrapperCommandSyntaxException(e);
+        }
+    }
+
+    public void expect(char c) throws WrapperCommandSyntaxException {
+        try {
+            stringReader.expect(c);
+        } catch (CommandSyntaxException e) {
+            throw new WrapperCommandSyntaxException(e);
+        }
+    }
```

Comment:
Maybe add JavaDocs to all these methods if this class aims to target developers?